platform/security/platform-security-core-api/src/main/java/ddf/security/samlp/SimpleSign.java

```diff
@@ -221,7 +222,12 @@ private PrivateKey getSignaturePrivateKey() throws SignatureException {
     }
 
     public boolean validateSignature(String queryParamsToValidate, String encodedSignature,
-            String encodedPublicKey) throws SignatureException {
+            @Nullable String encodedPublicKey) throws SignatureException {
+        if (encodedPublicKey == null) {
+            LOGGER.warn("Could not verify the signature of request because there was no signing certificate. Ensure that the IdP Metadata includes a signing certificate.");
+            return false;
```

Comment:
Is the last sentence necessary? I'm just wondering if there could ever be case where we'd want to validate a signature of a signing certificate not contained in IdP Metadata?